index.js

```diff
@@ -13,133 +13,167 @@
  * babel cli:
  *
  * babel [your-source-dir] --plugins=glamorous-to-emotion --presets=react,etc... --out-dir=[your-source-dir]
- * 
+ *
  * A demo can be seen at:
  * https://astexplorer.net/#/gist/7bc4771564a12c9f93c4904b3934aa1c/latests
  */
 
-module.exports = function (babel) {
-    const { types: t } = babel;
+module.exports = function(babel) {
+  const {types: t} = babel;
+
+  const fixContentProp = args => {
```

Comment:
What is the expected value of `args`?

Can we add a bit of documentation here so it's easier for people to understand? I'd like this project to be a bit educational as well: it helped you, so it ought to help others interested in AST transformation.

Reply:
Yes you're, right, I'll do a more descriptive pass soon :)